changelog.d/1-api-changes/finalise-v6

```diff
@@ -1 +1 @@
-Finalise version 6 and introduce new development version 7
+Finalise version 6 and introduce new development version 7 (#4179, ##)
```

Comment:
```suggestion
Finalise version 6 and introduce new development version 7 (#4179)
```

Or, does `##` have a special meaning? :thinking:

Reply:
## will put this PR's number when I squash it.

Reply:
Documented here: https://docs.wire.com/developer/developer/pr-guidelines.html#changelog-entries